AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/MySqlSettingRepository.cs

```diff
@@ -23,14 +24,16 @@ public class MySqlSettingRepository : ISettingRepository
         _settingsConverter = settingsConverter;
```

Comment:
Still need to inject the _settingGroupConverter here, similar to how it's already done for _settingsConverter